src/CPUSolver.h

```diff
@@ -44,6 +45,9 @@ class CPUSolver : public Solver {
   /** OpenMP mutual exclusion locks for atomic FSR scalar flux updates */
   omp_lock_t* _FSR_locks;
 
+public:
+  CPUSolver(TrackGenerator* track_generator=NULL);
```

Comment:
We already have this constructor - any idea why it is showing up as an addition in this diff?